server.js

```diff
@@ -204,6 +204,15 @@ const workspaceProxyOptions = {
             return 'not-matched';
         }
     },
+    onError: (err, req, res) => {
+        logger.err(`Error proxying workspace request: ${err}`);
```

Comment:
Dumb question, but is this `logger.err()` or `logger.error()`?

Reply:
😳